modules/blocksruntime/1.0.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/mackyle/blocksruntime/archive/9cc93ae2b58676c23fd02cf0c686fa15b7a3ff81.tar.gz",
+    "integrity": "sha256-vGehLJv98t1zzjldteGlKIOIwD37Yi/2cySXH539hro=",
```

Comment:
Ah so this is just from a specific commit. Then I think it would be better to change the version to something which reflects that.